docs/getting-started/nas/synology.md

```diff
@@ -91,10 +91,135 @@ This guide describes how to set up PhotoPrism using the new Synology user interf
 ### First Steps
 
 Our [First Steps 👣](../../user-guide/first-steps.md) tutorial guides you through the user interface and settings to ensure your library is indexed according to your individual preferences.
+-->
 
+## Setup using Docker and MariaDB
 
+### Prerequisites
+- Docker is installed (called 'Container Manager' to Synology.)
+- Create some folders for Photoprism to store configuration data and photos.
+  
+  ![Photoprism_1](./img/synology/Photoprism_1.jpg){ class="shadow" }
 
+- For testing purposes, add some pictures to your photos folder.
+- Later, if you're okay with your setup, you can link your pictures to the photos folder.
+
+### Preparing MariaDB
+- Launch `Container Manager` to access Docker
+- In the Registry tab, search for MariaDB
+  
+   ![Container Manager Sidebar showing Registry Tab](./img/synology/synology-registry.png){ class="shadow" }
+- Download and choose the flavor. (You may also skip ahead and grab `phpmyadmin` and `photoprism` while you're here or come back later.)
+  
+   ![Container Manager Registry Search Results for `mariadb`](./img/synology/synology-mariadb-serach.png){ class="shadow" }
+- Wait until you get the message your image is downloaded.
+- Switch to the `Container` tab and click `Create`
+  
+   ![Container Manager's Container Tab and Create Button](./img/synology/synology-create-container.png){ class="shadow" }
+- Choose the `mariadb:latest` (or whatever tag you chose as your image) then continue
+  
+   ![Create Container General Settings showing Image Name Dropdown](./img/synology/synology-choose-maria-image.png){ class="shadow" }
+- It is not necessary to add an external port mapping here in the `Port Settings` section as you can have your other container talk "internally" to this one, but you may want to expose a port so you can get at it from other services or administration tools later.
+  
+  ![Create Container Advanced Settings showing port mapping from inside container to outside world](./img/synology/synology-port-settings.png){ class="shadow" }
+- By default, when you scroll down, this will probably be attached to the `bridge` network with an IP address range of `172.17.0.0/16` or something similar. That's OK as long as you also attach the future Photoprism container to the same one.
+  
+  ![Create Container Advanced Settings showing network connection to the default `bridge` network in a dropdown with subnet and gateway details beneath](./img/synology/synology-network.png){ class="shadow" }
+- In the `Environment` section, click the `+ Add` button to add a new variable. Add `MYSQL_ROOT_PASSWORD` as the key and set the value to something secret. You can manipulate this later to eliminate or change this password, but you'll need it for now to get in there and make users.
```

Comment:
`MARIADB_ROOT_PASSWORD`. Per photo comment MARIADB_{USER,PASSWORD,DATABASE} here would eliminate the pma need.